# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../cinepaint/cinepaint-0.22-1-gcc43.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- cinepaint-0.22-1/plug-ins/bracketing_to_hdr/br_core/br_Image.cpp.orig	2011-04-14 12:26:35.804077826 +0200
+++ cinepaint-0.22-1/plug-ins/bracketing_to_hdr/br_core/br_Image.cpp	2011-04-14 12:27:39.216077825 +0200
@@ -26,6 +26,7 @@
 */
 #include <iostream>
 #include <cmath>                    // pow()
+#include <cstring>                  // strlen, strcpy
 
 #include "br_types.hpp"             // ImageID
 #include "br_enums.hpp"             // DataType, ReportWhat
--- cinepaint-0.22-1/plug-ins/bracketing_to_hdr/br_core/ResponseSolver.hpp.orig	2011-04-14 12:28:48.544077820 +0200
+++ cinepaint-0.22-1/plug-ins/bracketing_to_hdr/br_core/ResponseSolver.hpp	2011-04-14 12:29:25.904077820 +0200
@@ -34,6 +34,7 @@
 
 
 #include <fstream>
+#include <cstring>              // strlen
 #include "TNT/tnt_misc.hpp"     // proof_least_square_fit()
 #include "TNT/tnt_stopwatch.hpp"
 #include "TNT/jama_qr.hpp"      // JAMA::QR
--- cinepaint-0.22-1/plug-ins/pdf/pdf.cpp.orig	2011-04-16 00:29:21.884077833 +0200
+++ cinepaint-0.22-1/plug-ins/pdf/pdf.cpp	2011-04-16 00:30:52.296077813 +0200
@@ -37,6 +37,7 @@
 #define PLUG_IN_COPYRIGHT     "Copyright 2005-2006 Kai-Uwe Behrmann"
 
 
+#include <cstring>            // strlen, strcmp, strrchr
 #include "pdf.h"
 #include "pdf_dialog.h"
 
